pootle/core/views.py

```diff
@@ -646,26 +660,26 @@ def get_context_data(self, *args, **kwargs):
 class PootleTranslateView(PootleDetailView):
     template_name = "editor/main.html"
 
-    @property
-    def ctx_path(self):
-        return self.pootle_path
```

Comment:
because it was always being subclassed and coveralls showed it as not being used.

I think perhaps it should be copied out from one of the subclasses tho